index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Tribute Page</title>
+  <link rel="stylesheet" href="./styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+</head>
+
+<body>
+ <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+ <script src="js/index.js"></script>
+  <main id="main">
+    <h1 id="title">Dr. Norman Borlaug</h1>
+    <p class="text-center"><em>The man who saved a billion lives</em></p>
+    <figure id="img-div"><img id="image"
+        src="https://images.ctfassets.net/hrltx12pl8hq/7yQR5uJhwEkRfjwMFJ7bUK/dc52a0913e8ff8b5c276177890eb0129/offset_comp_772626-opt.jpg?fit=fill&w=800&h=300">
+      <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to
+        increase wheat yields - part of his life-long war on hunger. </figcaption>
+    </figure>
```

Comment:
Good one using the h1 tag for your title 👍 . Please consider wrapping the title with a header tag. The title in the header tag is the main description of the webpage. Using the header semantics increases the page accessibility by browsers  as search engines will be able to detect what the page is all about.

Reply:
Again, fantastic comment! Great job @Abenezer-Tilahun 🔥😎🔥 You even used emojis to make your review more friendly 😉